docs/mkdocs/docs/technical/contributing.md

```diff
@@ -34,7 +34,10 @@ $MY_PYTHON -m pip install -U pip setuptools wheel grpcio-tools
 ARCTIC_CMAKE_PRESET=skip $MY_PYTHON setup.py develop
 # Change the below Python_EXECUTABLE value to build against a different Python version
 cmake -DPython_EXECUTABLE=$MY_PYTHON -DTEST=off --preset linux-debug cpp
-cmake --build --preset linux-debug cpp
```

Comment:
This command is indeed using argument combinations not officially supported by `cmake`. Did it not work for you?

The more portable command is

```bash
cd cpp
cmake --build --preset linux-debug
```

Reply:
Indeed `cmake --build --preset linux-debug cpp` was not working on my machine.

```
cmake version 3.26.0
```

Do the proposed command (i.e. `cmake -S cpp --preset linux-release`) work for you or would you like to use the ones you have proposed?

Reply:
`cmake` without the `--build` parameter does the configure step. We want to do the build step at this point.